### DISCUSSION TITLE
Request for pipewire packages in Deepin

### DISCUSSION THREAD

**Discussion by prometheusAmb** [linuxdeepin/developer-center#4000] [Features Request & Ideas | 特性请求 & 头脑风暴]
Pipewire is the audio system many Linux distributions are moving towards. It offers many advantages, especially in terms of detecting and using Bluetooth devices. I would request native Deepin packages for pipewire, pipewire-media-session and pipewire-pulse (for pulseaudio compatibility for apps which require pulseaudio)

**Comment by BLumia** [comment]
cc @Zeno-sole @tsic404 @Decodetalkers

**Comment by xzl01** [comment]
Hello, I have tried using pipewire on deepin v23, and there is no problem so far, but a lot of testing work needs to be done before officially switching to pipewire. Maybe pipewire will only be provided as an optional solution for a period of time in the future.

**Reply by BLumia** [reply]
See https://github.com/linuxdeepin/developer-center/issues/3952 for the original report. I'm not sure but some packages like `pipewire-media-session` might need to be packaged first.

**Comment by Zeno-sole** [comment]
@prometheusAmb  You can package pipewire for deepin at https://github.com/deepin-community/
